frontend-service/build.gradle

```diff
@@ -36,6 +36,8 @@ dependencies {
     compile "io.swagger:swagger-annotations:${swaggerVersion}"
 
     compile "org.mapstruct:mapstruct-jdk8"
+
+    compileOnly "org.mapstruct:mapstruct-processor"
```

Comment:
Hm, is this needed along with the `apt` declaration on L41?

Reply:
See my comment in the PR summary, basically it's  "Add "compileOnly "org.mapstruct:mapstruct-processor" in the gradle to support IDE, so we can run the unit test for mapper in IDE. The IntelliJ doesn't support gradle apt well, it would not include the apt in the class path."

Reply:
Oh! I totally missed that, sorry.

If `compileOnly` works, do we actually gain anything by using the `apt` plugin? Should we just switch everything back to `compileOnly`?

Reply:
We still need `apt` for `gradle` to process the annotation process. The IntelliJ has different configuration for annotation processing. It's more like the  gradle + IntelliJ integration doesn't support `apt`  so far.

The IDE support for `apt` plugin is here: https://github.com/tbroyer/gradle-apt-plugin#intellij-idea   I haven't went though all of them, but `compileOnly` is the way from the sample config of this plugin.

Reply:
Yeah, I've got that too, but it's hit or miss whether the IDE actually applies it.

I did try just getting rid of the plugin completely in favor of just using `compileOnly/testCompileOnly` instead of `apt`, and that seemed to work... Let's come back to this later.